### ISSUE TITLE
run,run-local and run-remote not showing up from "make help"

### ISSUE THREAD

**Issue by tjyang** [hupe1980/scan4log4shell#4]
Is this intentional  ? 

- run,run-local and run-remote not showing up from "make help"
```
.PHONY: run
run:
	@go run *.go remote url -h

.PHONY: run-local
run-local:
	@docker-compose -f docker-compose.local.yml up --build

.PhONY: run-remote
run-remote:
	@docker-compose -f docker-compose.remote.yml up --build
```

**Comment by hupe1980** [comment]
Upps. This is not intentional. This is a bug

**Comment by tjyang** [comment]
Thanks for the confirmation. I like you Makefile. I correct that in my folk and added a [few more targets](https://github.com/learnmonitoring/scan4log4shell/blob/main/Makefile#L32) but extra is for my learning.

**Comment by hupe1980** [comment]
closed with #5
